### ISSUE TITLE
README example doesn't work

### ISSUE THREAD

**Issue by adelphes** [iarna/rtf-to-html#5]
Just running the basic example shown in your README doesn't work.

> $ npm install @iarna/rtf-to-html
> \+ @iarna/rtf-to-html@1.1.0
> added 12 packages in 1.829s
>
> $ cat index.js
```
const rtfToHTML = require('@iarna/rtf-to-html')

rtfToHTML.fromString('{\\rtf1\\ansi\\b hi there\\b0}', (err, html) => {
  console.log(html)
  // prints a document containing:
  // <p><strong>hi there</strong></p>
});
```

> $ node index.js
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <style>
    body {
      margin-left: 90pt;
      margin-right: 90pt;
      margin-top: 72pt;
      margin-bottom: 72pt;
      font-size: 12pt;
      text-indent: 0pt;
    }
    </style>
  </head>
  <body>
    
  </body>
</html>
```

There's no content inside the `<body>` tag, no matter what RTF you pass.

**Comment by cspmosh** [comment]
Getting the same as above. Using the example from the README, getting a empty body.

**Comment by rakaar** [comment]
Facing the same problem, I am unable to parse any other rtf text too. I used the same code as given in `README.md`.Is this solved?

**Comment by Romick2005** [comment]
Can we close this?

**Comment by iangilman** [comment]
It's still not working for me. I'm using the `npm` version, and it doesn't look like it has the fix.  I guess someone needs to do a fresh npm publish?

**Comment by OphtelDev** [comment]
The problem is in renderPara function:
it assumes that para is a Paragraph but it can be a Span too,

So I replaced line 82 with : 
if (!para.content || para.content.length === 0) return renderSpan(para, defaults)

And it works

Please fix it in the project

**Comment by iangilman** [comment]
Indeed. A similar fix is actually already on the master branch here:

https://github.com/iarna/rtf-to-html/blob/7f05d9d2b858e6949cb4491a95e6a5fc6250f59b/rtf-to-html.js#L47

But it hasn't been pushed to npm, so if you're using the npm version, it's broken. For my project, I just downloaded the code from here.

Hopefully someone involved in this library will push the latest version to npm!
